core/cloudflow-operator/src/main/scala/cloudflow/operator/CloudflowLabels.scala

```diff
@@ -55,4 +55,8 @@ object CloudflowLabels {
 
   // Managed by
   val ManagedByCloudflow = "cloudflow"
+
+  // Cloudflow installation namespace
+  val Namespace = "namespace"
+  val CloudflowNamespace = "cloudflow-namespace"
```

Comment:
These label key/values don't make much sense to me. what are they actually saying?

I think the purpose of this one specific label is to make the created ConfigMap more easily findable using a label query, e.g. the label key name should be based on the purpose of the `ConfigMap` and the value is essentially just the name of the config map itself as well.

So I suggest we remove the above two values and make both the key and value of the label to be `ProtocolVersionConfigMapName`

WDYT?

Reply:
yeah .. makes sense .. +1 .. will change